packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.module.scss

```diff
@@ -33,227 +45,359 @@
   color: $bpk-color-white;
   box-shadow: none;
 
-  @include breakpoint-landscape {
-    &__sponsor-label {
-      margin-bottom: bpk-spacing-sm();
+  @include bpk-breakpoint-small-tablet {
+    @include aspect-ratio(133%);
+  }
 
-      @include bpk-label-1;
-    }
+  @include bpk-breakpoint-mobile {
+    @include aspect-ratio(133%);
+  }
 
-    &__sponsor-content {
-      margin-bottom: bpk-spacing-xl();
+  @include bpk-breakpoint-small-mobile {
+    @include aspect-ratio(133%);
+  }
+
+  &::after {
+    display: none;
+  }
```

Comment:
I see this was already present before this change, but would it be possible to explain what this does?